setup.py

```diff
@@ -0,0 +1,52 @@
+from setuptools import setup, find_packages
```

Comment:
If you use PyCharm CE, then I would recommend using the `Code` | `Reformat Code` menu item.

Reply:
Same with other Python files in the repo.